mackerel-plugin-aws-rds/lib/metrics.go

```diff
@@ -74,13 +74,50 @@ func (p RDSPlugin) postgreSQLGraphDefinition() map[string]mp.Graphs {
 
 func (p RDSPlugin) auroraGraphDefinition() map[string]mp.Graphs {
 	return map[string]mp.Graphs{
+		p.Prefix + ".CPUUtilization": {
```

Comment:
[memo]
Definitions of those "general" metrics are also exists in `p.baseGraphDefs()`,  but we cannot simply merge it with original `p.auroraGraphDefinition()` because there're some definitions which exists in both `auroraGraphDefinition()` and `baseGraphDefs()` but each has different metrics (like `Throughput`)